test/EmptyApp.FunctionalTests/project.json

```diff
@@ -4,23 +4,18 @@
   "buildOptions": {
     "keyFile": "../../keys/35MSSharedLib1024.snk",
     "delaySign": true,
-    "preserveCompilationContext": true
+    "preserveCompilationContext": true,
+    "nowarn": [ "CS0618" ]
   },
   "dependencies": {
-    "dotnet-test-xunit": "1.0.0-*",
+    "dotnet-test-xunit": "2.2.0-*",
     "FunctionalTestUtils": { "target": "project" },
     "Microsoft.AspNetCore.Diagnostics": "1.0.0",
-    "Microsoft.NETCore.Platforms": "1.0.1",
-    "xunit": "2.1.0"
+    "xunit": "2.2.0-*"
```

Comment:
I think for xunit it is just a notification, not an approval. But still needs to be filed. @dnduffy will you pls do it?